### PR TITLE
[REFACTOR] 계획 완료 시 조회되는 목록 로직 수정

### PR DESCRIPTION
**Pull request by 5hseok**
## #️⃣ Related Issue

Linked Feature backlog: 

## 📝 Purpose of PR

Please briefly explain your work in this PR.

## Contents

계획을 완료하면 completedDate가 가장 오래된 목록 2개와 방금 완료한 계획까지 총 3개를 보여주도록 수정했습니다. 

## Checklist

-   [ ] Does commit messages follow the convention?
-   [ ] Are variable names brief but descriptive?
-   [ ] Is the code clean and easy to understand?
-   [ ] If a new feature has been added, or a bug fixed, has a test been added to confirm good behavior?
-   [ ] Does the test successfully test edge and corner cases?

## 💬 Review Requirements (Optional)

Is there anything you can praise about this PR? Start with praise.
Make a review, leaving kind comments and suggesting changes where needed (to resolve the above).

### CODE REVIEW

File: src/main/java/ac/dnd/dodal/application/plan/repository/PlanRepository.java [hunk 1]
```diff
@@ -64,4 +64,13 @@ boolean isExistByUserIdAndGoalIdAndHistoryId(
             + "AND goal.deletedAt IS NULL "
             + "AND goal.userId = :userId")
     boolean isExistByUserIdAndGoalId(@Param("userId") Long userId, @Param("goalId") Long goalId);
+
+    @Query("SELECT p FROM plans p "
+          + "WHERE p.history.historyId = :historyId "
+          + "ORDER BY p.completedDate ASC "
+          + "LIMIT 2 "
+          + "UNION "
```
Comment (5hseok) [inline src/main/java/ac/dnd/dodal/application/plan/repository/PlanRepository.java:72]: 중복되는 것은 하나만 나올 수 있게 UNION으로 DISTINCT 기능을 수행합니다.

File: src/main/java/ac/dnd/dodal/application/plan/service/PlanCommandService.java [hunk 2]
```diff
@@ -96,10 +97,10 @@ public void createPlanAndHistory(CreatePlanAndHistoryCommand command) {
 
     // TODO: MVP 배포 후 직접 입력 받는 것으로 수정
     @Override
-    public Plan completePlan(CompletePlanCommand command) {
+    public List<PlanElement> completePlan(CompletePlanCommand command) {
         Plan plan = planService.findByIdOrThrow(command.planId());
         List<PlanFeedback> feedbacks = new ArrayList<>();
-        // PlanFeedback feedback = new PlanFeedback(command.question(), command.indicator());
+//         PlanFeedback feedback = new PlanFeedback(command.question(), command.indicator());
```
Comment (5hseok) [inline src/main/java/ac/dnd/dodal/application/plan/service/PlanCommandService.java:103]: 여기를 주석을 풀고 해야할 지, 아래에 있는 코드로 해야할 지 모르겠어요
